build.sbt

```diff
@@ -15,12 +15,6 @@ val Scala213 = "2.13.8"
 ThisBuild / crossScalaVersions := Seq(Scala213, "2.12.15", "3.0.2")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-ThisBuild / homepage := Some(url("https://github.com/TimWSpence/cats-stm"))
-
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/TimWSpence/cats-stm"), "git@github.com:TimWSpence/cats-stm.git")
-)
```

Comment:
This is set automatically. Actually with more info then you've set here I believe 😆